src/scss/_objects.about.scss

```diff
@@ -0,0 +1,13 @@
+.about-page--team-member {
```

Comment:
hey we try to avoid adding custom styling for pages. Do you think there is a way to solve that without the need to add custom css for it?